package.json

```diff
@@ -39,17 +38,26 @@
     "@types/react-dom": "^17.0.0",
     "@types/styled-components": "^5.1.2",
     "ganache-core": "2.12.1",
-    "husky": "^7.0.2",
     "prettier": "^2.3.1",
     "react-app-rewired": "^2.1.6",
     "react-scripts": "3.4.0",
-    "typescript": "~4.1.3"
+    "typescript": "~4.1.3",
+    "@zodiac-module/reality": "github:gnosis/zodiac-module-reality#main",
+    "@zodiac-module/exit": "github:gnosis/zodiac-module-exit#carlos/packages",
+    "@zodiac-module/delay": "github:gnosis/zodiac-modifier-delay#carlos/ethers-version",
```

Comment:
🛑  We need to merge these PRs first:
- Delay Modifier: https://github.com/gnosis/zodiac-modifier-delay/pull/23
- Exit Module: https://github.com/gnosis/zodiac-module-exit/pull/44

Reply:
Done!